src/EventSubscriber/CreateNewWishlistSubscriber.php

```diff
@@ -11,60 +11,35 @@
 namespace BitBag\SyliusWishlistPlugin\EventSubscriber;
 
 use BitBag\SyliusWishlistPlugin\Entity\WishlistInterface;
-use BitBag\SyliusWishlistPlugin\Entity\WishlistToken;
-use BitBag\SyliusWishlistPlugin\Factory\WishlistFactoryInterface;
-use BitBag\SyliusWishlistPlugin\Repository\WishlistRepositoryInterface;
-use BitBag\SyliusWishlistPlugin\Resolver\TokenUserResolverInterface;
 use BitBag\SyliusWishlistPlugin\Resolver\WishlistCookieTokenResolverInterface;
 use BitBag\SyliusWishlistPlugin\Resolver\WishlistsResolverInterface;
-use Sylius\Component\Channel\Context\ChannelContextInterface;
-use Sylius\Component\Channel\Context\ChannelNotFoundException;
-use Sylius\Component\Core\Model\ChannelInterface;
-use Sylius\Component\Core\Model\ShopUserInterface;
 use Symfony\Component\EventDispatcher\EventSubscriberInterface;
 use Symfony\Component\HttpFoundation\Cookie;
-use Symfony\Component\HttpFoundation\Response;
+use Symfony\Component\HttpFoundation\RequestStack;
 use Symfony\Component\HttpKernel\Event\RequestEvent;
 use Symfony\Component\HttpKernel\Event\ResponseEvent;
 use Symfony\Component\HttpKernel\KernelEvents;
-use Symfony\Component\Security\Core\Authentication\Token\Storage\TokenStorageInterface;
 
 final class CreateNewWishlistSubscriber implements EventSubscriberInterface
 {
     private string $wishlistCookieToken;
 
     private WishlistsResolverInterface $wishlistsResolver;
 
-    private WishlistFactoryInterface $wishlistFactory;
-
-    private WishlistRepositoryInterface $wishlistRepository;
-
-    private TokenStorageInterface $tokenStorage;
-
-    private ChannelContextInterface $channelContext;
-
     private WishlistCookieTokenResolverInterface $wishlistCookieTokenResolver;
 
-    private TokenUserResolverInterface $tokenUserResolver;
+    private RequestStack $requestStack;
 
     public function __construct(
         string $wishlistCookieToken,
         WishlistsResolverInterface $wishlistsResolver,
-        WishlistFactoryInterface $wishlistFactory,
-        WishlistRepositoryInterface $wishlistRepository,
-        TokenStorageInterface $tokenStorage,
-        ChannelContextInterface $channelContext,
         WishlistCookieTokenResolverInterface $wishlistCookieTokenResolver,
-        TokenUserResolverInterface $tokenUserResolver,
+        RequestStack $requestStack,
```

Comment:
If there's "php": "^8.0", wouldn't it be better to use the new way of writing a constructor with private Service $service?

Reply:
With this PR I don't want to change the code style. I'm going to do it with next PRs.